google/cloud/spanner/samples/samples.cc

```diff
@@ -3802,6 +3802,16 @@ void QueryInformationSchemaDatabaseOptions(
 }
 // [END spanner_query_information_schema_database_options]
 
+// [START spanner_set_batching_delay]
+void CommitWithBatchingDelay(google::cloud::spanner::Client client,
+			     google::cloud::spanner::Mutations mutations,
+			     absl::Duration batching_delay) {
+  google::cloud::spanner::CommitOptions options;
```

Comment:
`CommitOptions` is a deprecated part of the public API.  Whatever is needed here should be expressed in terms of `google::cloud::Options`, and then by calling another `Commit()` overload.

Reply:
I **think** I've done this. This field is much like RequestPriorityOption, and I've pretty much copied it. I don't think we need to overload Commit() again, as the current versions all take an Options field. Please take a look and let me know if I've messed this up.

Reply:
All the current overloads take an `Options` argument, not a `CommitOptions`.  The latter is deprecated, and we shouldn't be changing it.  Whatever is needed here should **only** be expressed in terms of `google::cloud::Options`.

Reply:
I'm sorry ... my "only" was too strict.  We still need your modifications to `CommitOptions` as they are used internally (and for mocking).  But we don't want to use `CommitOptions` as part of any sample.  Instead, the sample should use `MaxBatchingDelayOption` directly.